tests/Unit/PointwiseFunctions/GeneralRelativity/Test_Tags.cpp

```diff
@@ -7,9 +7,52 @@
 
 #include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
 
-struct DataVector;
+namespace {
+struct ArbitraryFrame;
+struct ArbitraryType;
+}  // namespace
+
+#define _TEST_SIMPLE_TAGS(D, F, T)                                            \
```

Comment:
Same comment about making this a function template

Reply:
changed